### PR TITLE
Add pdf uploader functionality

### PR DESCRIPTION
**Pull request by kirch1**
### Files Changed:
- process.tsx
- apicalls.tsx
### What Has Changed: 
- send request to back-end via pdf file instead of raw text

### CODE REVIEW

File: src/Components/Process/Process.tsx [hunk 1]
```diff
@@ -104,14 +120,17 @@ const Process = ({tosInput, setTosInput, setConcerns, setError, user}: FormProps
       <form className="form-card">
         {loading ? (
           <>
-            <h3 className="form-heading"> Processing Terms of Service, Please Wait</h3>
+            <h3 className="form-heading">Processing Terms of Service, Please Wait</h3>
```
Comment (chrissycooper) [inline src/Components/Process/Process.tsx:123]: nice attention to detail

File: src/Components/Process/Process.tsx [hunk 2]
```diff
@@ -123,15 +142,20 @@ const Process = ({tosInput, setTosInput, setConcerns, setError, user}: FormProps
...
                 <Select
                   labelId="tos-library-label"
                   id="tos-library-select"
                   value={selectedLibrary}
                   label="Select TOS"
-                  onChange={handleLibraryChange}>
-                  {tosLibrary.map(tos => <MenuItem key={tos.service} value={tos.tos}>{tos.service}</MenuItem>)}
+                  onChange={handleLibraryChange}
+                >
+                  {tosLibrary.map((tos) => (
```
Comment (chrissycooper) [inline src/Components/Process/Process.tsx:154]: These are really nice adjustments, it makes this block of code a lot more readable

File: src/Components/Process/Process.tsx [hunk 3]
```diff
@@ -141,14 +165,28 @@ const Process = ({tosInput, setTosInput, setConcerns, setError, user}: FormProps
...
-                > Process
-                </Button>
-                <Button
-                  color="primary"
-                  variant="outlined"
-                  startIcon={<UploadFileIcon />}> Upload
-                </Button>
+                  startIcon={<CheckRoundedIcon />}>Process</Button>
+                <input
+                  disabled={!getConcernCount()}
```
Comment (chrissycooper) [inline src/Components/Process/Process.tsx:170]: beautiful error handling

**Review by Jesusborjas006 — APPROVED** [review]
Both functions are looking good sending pdf file to the server!🔥

**Review by chrissycooper — APPROVED** [review]
Looks great, really excited that this functionality is up and running!

**Pull request merged**